governance-exercises/JanSky Polling Exercise 4 - Forum Post.md

```diff
@@ -0,0 +1,20 @@
+# Forum Response
+
+Author: JanSky  
+Date: Wednesday 9 August 2023
+
+---
+
+This situation requires an immediate response. A couple of recommendations that should happen promptly:
+
+**Emergency Poll on Fix A**: I will be initiating an emergency poll presently to see if there is community support for Fix A. **This poll will only last 12 hours, @Aligned Delegates please vote as soon as you are ready and able.** While [MIP24](https://mips.makerdao.com/mips/details/MIP24#MIP24c3) states that a poll might not be strictly necessary in emergencies, I am recommending it given the difference of opinion between Roundtable and WeCodeCrypto.
+
+**Emergency Governance Call**: I am organizing a governance call with Roundtable Develop, WeCodeCrypto, and Imogen. @Ecosystem Facilitator should also attend if they are able (please message me on Discord for information, you have a message there in your inbox). I will summarize and release notes from this call to the community once this Emergency Response stops.
```

Comment:
Yeah, as noted on summary contents, this is best decided privately before bringing it up publicly. Basically, Imogen has likely already been in a call about it, it was unclear in the exercise, but likely you've been in the call prior to this point as well.

Reply:
Basically, its important that you don't try to take up the coordination role in an emergency if someone is already doing it without speaking to them first. This has the potential to confuse everyone, or start a pissing match at the worst possible time. 

As the author of this post, its assumed that Imogen is in the coordination role.

Reply:
As a minor note, I would also not call this a governance call. 'Governance Call' implies that there will be representatives of governance (ie delegates or MKR holders) there. Usually War room / Coordination . etc